documentation/flows/agent/dynamic_backhaul_switching.md

```diff
@@ -0,0 +1,113 @@
+
+# Introduction
+
+Backhaul Switching is the process where the Repeater / Extender switches from one backhaul link to another.
+It can be triggered by any of the following:
+1.	Controller backhaul steering request as part of backhaul optimization process ([1] section 12)
+2.	Wired backhaul link up/down events – wired link down will trigger backhaul switch to the wireless link, whereas wired link up will trigger a backhaul switch to the wired link.
+Future enhancement for selecting the better link (based on phy rate) is out of scope of this document.
+3.	UCC commands – for certification, dev_reset_default should reset any previous onboarding state. This means disabling the backhaul, whichever it is. Later, when a dev_set_config is received, switching to the requested backhaul type will take place.
+
+## Current state
```

Comment:
You are right, but this should be updated in the implementation PR - otherwise it will be too confusing for the implementor (which will not be me IMO)